### ISSUE TITLE
'Upgrading Flutter' page issue

### ISSUE THREAD

**Issue by chalin** [flutter/website#1885]
Page URL: https://flutter.io/docs/development/tools/sdk/upgrading.html
Page source: https://github.com/flutter/website/tree/master/src/docs/development/tools/sdk/upgrading.md

Two related issues:

- This page contains a mention of flutter branches, yet the Flutter CLI command is `flutter channel`. So the "channel" terminology should be used.
- The page starts with "We strongly recommend tracking the stable branch...". IMHO, such text would be more appropriate as part of the Archive page. Consider moving the text.

@sfshaza2 WDYT?

**Comment by kwalrath** [comment]
Agreed, the conflation of branches and channels seems bad. We should stick to channels, wherever possible, to mirror the command.
